pgdb/api.py

```diff
@@ -151,9 +151,6 @@ def list(self, request, *args, **kwargs):
             else:
                 corpora = models.Corpus.objects.filter(user_permissions__user=request.user).all()
 
-        for c in corpora:
-            c.generate_enrichments()

```

Comment:
Also should remove the `Corpus.generate_enrichments` function: https://github.com/MontrealCorpusTools/polyglot-server/blob/master/pgdb/models.py#L547